requirements.txt

```diff
@@ -1,4 +1,4 @@
 black==19.3b0
 gunicorn==19.9.0
 pytest-flake8==1.0.4
-pytest==4.2.1
+pytest==5.2.0
```

Comment:
Was facing an issue where pytest would fail to run. Looks like it's linked to this issue: https://github.com/pytest-dev/pytest/issues/3280

The new version is the latest released.